source/index.html.md

```diff
@@ -630,6 +770,35 @@ object][ApplePay-PaymentToken] for more information.
 </p>
 
 
+##### Method: `samsungpay`
+
+Samsung Pay provides the payment details as either JWE or JWE/JWS JOSE objects.
+Currently only the JWE is accepted.
+
+<dl class="dl-vertical">
+  <dt>
+    samsungpay[jwt]
+    <span class="type">
+      <a href="https://tools.ietf.org/html/rfc7516#section-3.1">
+        Compact serialized JWE token
+      </a>
+    </span>
+  </dt>
+  <dd>
+    Complete JWE token as received from Samsung Pay.
+  </dd>
+  <dt>
+    samsungpay[cek]
+    <span class="type">
+      [:base64:]{24}
+    </span>
+  </dt>
+  <dd>
+    The 128 bit symmetric AES content encryption key (CEK) used to encrypt the
```

Comment:
Tried to find mention of the name "content encryption key" in Samsung Pay documentation 🤞 , but 😞
The Web Checkout Integration Guide mentions `plainEncKey` but 🚫

Perhaps `symmetric_key` would be as good as `cek`? And then at least it'd align with https://github.com/clearhaus/gateway-api-docs/pull/97/files#diff-d30d91dbcd59693b69f9242acb4630c5R683 🙁

Reply:
So, I agree that `symmetric_key` is accurate. But since this is a JWE, I'd like to be explicit about what exactly this is in the terms of a JWE. Here it is very explicit, since a JWE could technically also contain a second symmetric key.

(If you ask me, the Samsung guides use bad naming)